### ISSUE TITLE
Provide a dilated value for testing

### ISSUE THREAD

**Issue by huntc** [akka/alpakka-kafka#1118]
I believe we need a way of providing a `dilated` as per Akka test kits so that we can accommodate timings on CI. Here's the line in question that we must override in every test:

https://github.com/akka/alpakka-kafka/blob/d91c72fb2b1b339a350d52de55f864b674ef89fc/testkit/src/main/java/akka/kafka/testkit/javadsl/BaseKafkaTest.java#L62

**Comment by seglo** [comment]
@huntc I agree that would be useful.  The Alpakka Kafka JUnit testkit doesn't subclass or encapsulate an Akka testkit like the Alpakka Kafka Scala testkit does.  We could explore changing that.  In the meantime, if your tests are already using a testkit maybe you could add some convenience methods to a base test class.

```java
  TestKit testKit = new TestKit(system);

  protected <T> T resultOf(CompletionStage<T> stage) throws Exception {
    return resultOf(stage, testKit.dilated(testKit.getRemainingOrDefault()));
  }

  protected <T> T resultOf(CompletionStage<T> stage, Duration timeout) throws Exception {
    return stage.toCompletableFuture().get(timeout.toMillis(), TimeUnit.MILLISECONDS);
  }
```

**Comment by huntc** [comment]
Thanks for the suggestion. We will likely add something that to many tests.
